panels/source/arrangers/OtherArrangers.js

```diff
@@ -45,9 +45,12 @@ enyo.kind({
 			var f = this.container.toIndex;
 			var c$ = this.getOrderedControls(f);
 			var o = Math.floor(c$.length/2);
+		        var wf = (s === 0 && f == (c$.length-1));
```

Comment:
when you say reversed, you mean? Line numbers? 

wf "wrap forward", wb "wrap backward" determines if 0 panel to max panel, or max panel to 0 panel flow happened.